hal/tests/TESTS/mbed_hal/flash/functional_tests/CMakeLists.txt

```diff
@@ -3,11 +3,16 @@
 
 cmake_minimum_required(VERSION 3.19.0 FATAL_ERROR)
 
-set(MBED_PATH ${CMAKE_CURRENT_SOURCE_DIR}/../../../../.. CACHE INTERNAL "")
+set(MBED_PATH ${CMAKE_CURRENT_SOURCE_DIR}/../../../../../.. CACHE INTERNAL "")
 set(TEST_TARGET mbed-hal-flash-functional-tests)
 
 include(${MBED_PATH}/tools/cmake/mbed_greentea.cmake)
 
 project(${TEST_TARGET})
 
-mbed_greentea_add_test(TEST_NAME ${TEST_TARGET})
+mbed_greentea_add_test(
```

Comment:
This change here shouldn't be part of "CMake: fix incorrect relative path for MBED_PATH". It should be part of "CMake: Require TEST_SOURCES in greentea CMake file" instead.

Reply:
Ah snap I did not catch that! Done now 👍